alvr/server_core/src/connection.rs

```diff
@@ -7,8 +7,7 @@ use crate::{
     tracking::{self, TrackingManager},
     ConnectionContext, ServerCoreEvent, ViewsConfig, FILESYSTEM_LAYOUT, SESSION_MANAGER,
 };
-use alvr_adb::wired_connection::{WiredConnection, WiredConnectionStatus};
-use alvr_audio::AudioDevice;
```

Comment:
Doesn't this ago against your usual import directive?

Reply:
I don't understand what you mean

Reply:
Well your directive is to always import a struct and then not have to specify it later, you changed that here

Reply:
ah yeah. it't to remove the unused import lint